projects/ccf-database/src/lib/hubmap/hubmap-data.ts

```diff
@@ -326,7 +326,10 @@ export class HuBMAPTissueBlock {
   }
 
   getDatasetThumbnail(dataset: JsonDict, assetsApi: string, serviceToken?: string): string | undefined {
-    if (dataset.group_uuid === '73bb26e4-ed43-11e8-8f19-0a7c1eab007a') { // TMC-Vanderbilt
+    if (dataset.thumbnail_file) {
+      const uuid = (dataset.thumbnail_file as JsonDict).file_uuid;
+      return `${uuid}/thumbnail.jpg` + (serviceToken ? `?token=${serviceToken}` : '');
```

Comment:
Don't assume the filename is thumbnail.jpg. use `datasets.thumbnail_file?.filename` to get the filename.